main.py

```diff
@@ -71,19 +64,30 @@ def showMap(city):
 
 @app.route('/', methods=['GET', 'POST'])
 def index():
-    cityName = getRandomCityName(POPULATION_THRESHHOLD)
-    cityCoords = getCityCoordinatesByName(cityName)
-    city = City(cityName, cityCoords)
+    city = getRandomCity()
     mapOfCity = showMap(city)
+    correctAnswer = city.cityId
+
+    return render_template('index.html', mapOfCity=mapOfCity, correctAnswer=correctAnswer)
+
+@app.route('/guess', methods=['POST'])
+def checkAnswer():
+    req = request.get_json()
     
-    return render_template('index.html', mapOfCity=mapOfCity)
-    
+    if request.method == 'POST':
```

Comment:
Is this not guaranteed to be a POST request because of line 73 saying `methods=['POST']`? You could reduce indentation by 1 layer